net/openthread-br/Makefile

```diff
@@ -48,31 +48,22 @@ define Package/openthread-br/description
   A Thread border router for POSIX-based platforms.
 endef
 
-define Package/openthread-br/conffiles
-/var/lib/thread
-endef
```

Comment:
Please keep this so that the content of /var/lib/thread is kept during sysupgrade.